discord/bot.py

```diff
@@ -346,7 +346,11 @@ async def register_commands(self) -> None:
                     )
                     raise
 
-    async def process_application_commands(self, interaction: Interaction) -> None:
+    async def process_application_commands(
+            self,
+            interaction: Interaction,
+            application_context_cls: Optional[Type[ApplicationContext]] = None
```

Comment:
Looks like some code from #424 got thrown in here.

Reply:
Lol

Reply:
idk how it even came into this branch, but it should be solved now